index.html

```diff
@@ -143,6 +141,22 @@ <h2>About <br> Myself</h2>
 
                 </div>
         </section>
+        <section class="contact-form">
+            <div class="first-container">
+                <h2>Contact me </h2>
+                <p>If you have an application you <br> are interested in developing, <br> a feature that you need built or a <br> project that needs coding. <br> I’d love to help with it </p>
+            </div>
+            <form action="https://formspree.io/f/mzboldye" method="post">
+                <ul>
+                    <li><input type = "text" id="name"  name="user_name" maxlength = "30" placeholder="Yeremias"  required></li>
+                    <li><input type="email" id="email" name="user_email" placeholder="natajayanj14@gmail.com" required></li>
```

Comment:
- [ ] Kindly write in the placeholder a sentence that helps the user to know what to write like `Your email` but `natajayanj14@gmail.com` makes the user confused about what to write.